src/_node/node/actions.ts

```diff
@@ -787,17 +787,17 @@ const edit = ({
     });
 
     // predict needToSelectNodePaths
-    const needToSelectNodePaths = (() => {
-      const needToSelectNodePaths: string[] = [];
-      const validNodeTree = getValidNodeTree(nodeTree);
-      const focusedNode = validNodeTree[targetUid];
-      const parentNode = validNodeTree[focusedNode.parentUid as TNodeUid];
-      const focusedNodeChildIndex = getNodeChildIndex(parentNode, focusedNode);
-      const newNodePath = `${parentNode.data.path}${NodePathSplitter}${focusedNode.data.tagName}-${focusedNodeChildIndex}`;
-      needToSelectNodePaths.push(newNodePath);
-      return needToSelectNodePaths;
-    })();
-    dispatch(setNeedToSelectNodePaths(needToSelectNodePaths));
+    // const needToSelectNodePaths = (() => {
+    //   const needToSelectNodePaths: string[] = [];
```

Comment:
But we need to know which node to select. Let's say we copy and paste so we need to focus the pasted node as they are newly added node so I suspect we can't remove this method. It has to be triggered conditionally

Reply:
ok got it so this is for the case when we edit the node. Yeah it is commented so it's good